powerspikegg/rawdata/tools/fetch/summoner.go

```diff
@@ -0,0 +1,132 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"strings"
+	"io"
+	"errors"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/google/subcommands"
+
+	fetcherpb "powerspike.gg/powerspikegg/rawdata/fetcher/service_gopb"
+	lolpb "powerspike.gg/powerspikegg/rawdata/public/leagueoflegends_gopb"
+)
+
+type summonerCommand struct {
+	base
+
+	// flags
+	regionFlag string
+
+	// parsed flags / arguments
+	summonerName string
+	region lolpb.Region
+
+	// TODO(ArchangelX360): implement update by summonder ID
+}
+
+
+// registerSummonerCommand create a new summoner command
```

Comment:
If you want to break this comment line (I like to do so, but it is not a requirement in the style guides), break it right before the 80 char limit:

```
// registerSummonerCommand create a new summoner command and registers it into
// subcommands.